ArduSub/motors.cpp

```diff
@@ -130,11 +130,111 @@ void Sub::motors_output()
 {
     // check if we are performing the motor test
     if (ap.motor_test) {
-        return; // Placeholder
+        verify_motor_test();
+    } else {
+        motors.set_interlock(true);
+        motors.output();
     }
-    motors.set_interlock(true);
-    motors.output();
 }
+static uint32_t last_do_motor_test_fail_ms = 0;
+static uint32_t last_do_motor_test_ms = 0;
```

Comment:
Maybe you copied this from another vehicle, but can we change this to the Sub class?

Reply:
Whoops. yes.